test/esp/integration/alert_integration_test.rb

```diff
@@ -0,0 +1,96 @@
+require File.expand_path(File.dirname(__FILE__) + '/../../test_helper')
+
+module ESP::Integration
+  class AlertTest < ESP::Integration::TestCase
+    context ESP::Alert do
+      context 'live calls' do
+        setup do
+          @report = ESP::Report.all.detect { |r| r.status == 'complete' }
+          skip "Live DB does not have any reports.  Add a report and run tests again." if @report.blank?
```

Comment:
I'm not sure.  I skipped it mainly for local dev runs so you wouldn't get a bunch of failures just because the dev data is not the same as the tests expect.  That may not be so necessary now that `rake test` does not run the live tests.  You have to run them explicitly with `rake test:all` or `rake test:integration`.  Do you think I should make them all fails?